db/migrate/20171016143522_create_subscribers.rb

```diff
@@ -0,0 +1,8 @@
+class CreateSubscribers < ActiveRecord::Migration[5.1]
+  def change
+    create_table :subscribers do |t|
+      t.string :address
```

Comment:
Would it make sense to make this `null: false` and add a unique index to this column?

Reply:
Yes, probably but again them needing to be unique hasn't been 'established' yet (although it's obviously very unlikely we won't want them to be). Trying not to get to far ahead of 'minimal'.

Reply:
[Trello](https://trello.com/c/BhKdFEXN/170-subscriber-validation)